redfish-core/lib/managers.hpp

```diff
@@ -111,15 +114,25 @@ inline void
     VariantType dbusPropertyValue(propertyValue);
 
     crow::connections::systemBus->async_method_call(
-        [asyncResp](const boost::system::error_code ec) {
+        [asyncResp, req](const boost::system::error_code ec) {
             // Use "Set" method to set the property value.
+            std::string postPath =
+                "op=POST:/redfish/v1/Managers/bmc/Actions/Manager.Reset/";
             if (ec)
             {
                 BMCWEB_LOG_DEBUG << "[Set] Bad D-Bus request error: " << ec;
                 messages::internalError(asyncResp->res);
+                audit::audit_post_event(
+                    AUDIT_USYS_CONFIG, postPath, req.session->username.c_str(),
+                    boost::asio::ip::host_name().c_str(),
+                    req.ipAddress.to_string().c_str(), NULL, false);
                 return;
             }
 
+            audit::audit_post_event(
+                AUDIT_USYS_CONFIG, postPath, req.session->username.c_str(),
+                boost::asio::ip::host_name().c_str(),
+                req.ipAddress.to_string().c_str(), NULL, true);
```

Comment:
The only difference between this one and the one above is the false/true ?

Reply:
right, it is the value that indicates if an event was successful or not. I was thinking, there might be a case where we would want to log when a user without particular clearance attempted a call or just a failed attempt. However, if just logging the successful calls was important, we could do away with the changes to the doBMCGracefulRestart that allow access to req.